chaste_codegen/data/tests/all_models_reference_models/BE/Shannon2004.cpp

```diff
@@ -67,7 +67,7 @@
         this->mParameters[8] = 16.0; // (var_INa__G_INa) [milliS_per_microF]
         this->mParameters[9] = 0.029999999999999999; // (var_IKr__G_IKr_const) [milliS_per_microF]
         this->mParameters[10] = 0.070000000000000007; // (var_IKs__G_Ks_Max) [milliS_per_microF]
-        this->mParameters[11] = 9.0; // (var_INaCa__V_max) [microA_per_microF]
+        this->mParameters[11] = 0; // (var_INaCa__V_max) [microA_per_microF]
```

Comment:
This change worries me!

Reply:
It's always worth sanity-checking the diff for a few generated files to see if things like this crop up!

Reply:
hhm yeah I checked most but not all of them, quite a lot of fukes to check so thanks for spotting it

Reply:
should be fixed due to your suggested direction change